sdk/resourcemanager/azure-resourcemanager-appservice/src/main/java/module-info.java

```diff
@@ -0,0 +1,14 @@
+module com.azure.resourcemanager.appservice {
+    requires transitive com.azure.resourcemanager.keyvault;
+    requires transitive com.azure.resourcemanager.msi;
+    requires transitive com.azure.resourcemanager.dns;
+    requires transitive com.azure.resourcemanager.storage;
```

Comment:
Do these have to be transitive? Will the consumers of `azure-resourcemanager-appservice` module use the types defined in keyvault, msi, dns and storage modules?